api/app_analytics/views.py

```diff
@@ -32,6 +40,59 @@
 logger = logging.getLogger(__name__)
 
 
+class SDKAnalyticsFlagsV2(CreateAPIView):
+    permission_classes = (EnvironmentKeyPermissions,)
+    authentication_classes = (EnvironmentKeyAuthentication,)
+    serializer_class = SDKAnalyticsFlagsSerializer
+
+    def create(self, request: Request, *args, **kwargs) -> Response:
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        self.evaluations = serializer.validated_data["evaluations"]
+        if not self._is_data_valid():
+            # Mirror v1 implementation and return 200 to avoid
+            # breaking client integrations.
```

Comment:
Since we're creating a new endpoint here, we can probably change this if we want to?

Reply:
Ok I've changed the name and sent a HTTP `400` status code.